_core/regist/regist.php

```diff
@@ -117,9 +120,9 @@ private function insert($info) {
         $keys = implode(",",$keys);
         $vals = implode(",",$vals);
 
-        $query = "insert into ".SQLLOG." ($keys) values ($vals)";
+        $query = "INSERT INTO ".SQLLOG." ($keys) VALUES ($vals) ON DUPLICATE KEY UPDATE no = no+1";
         $mysql->query($query);
-        $final = (int) $mysql->result('select last_insert_id()');
+        $final = (UNIFIED_TABLE) ? $mysql->result("SELECT MAX(no) FROM " . SQLLOG . " WHERE board='" . BOARD_DIR . "'") : (int) $mysql->result('SELECT last_insert_id()');
```

Comment:
From my looking around, `last_insert_id()` [only works on `AUTO INCREMENT`'ed fields](http://php.net/manual/en/function.mysql-insert-id.php#refsect1-function.mysql-insert-id-description), which for the post table to work, is not auto-incremented